src/seedu/addressbook/commands/LengthCommand.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.commands;
+
+public class LengthCommand extends Command {
+    public static final String COMMAND_WORD = "length";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Returns the current length of the address book at the point of query.\n\t"
+            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_SUCCESS = "Length of the address book is: ";
+
+    @Override
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.